test/images/agnhost/resource-consumer/consume-cpu/main.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package consumecpu
+
+import (
+	"math"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+// CmdConsumeCPU is used by agnhost Cobra.
+var CmdConsumeCPU = &cobra.Command{
+	Use:   "consume-cpu",
+	Short: "TBA",
+	Long:  "TBA",
```

Comment:
ditto: `Consumes the given amount of CPU over the given amount of time. This command is also used by the \"resource-consumer\" subcommand.`